deploy/olm-catalog/ibm-management-ingress-operator/1.11.0/ibm-management-ingress-operator.v1.11.0.clusterserviceversion.yaml

```diff
@@ -98,7 +98,7 @@ spec:
   relatedImages:
     - name: ICP_MANAGEMENT_INGRESS_IMAGE
       image: quay.io/opencloudio/icp-management-ingress:2.5.14
-    - name: ibm-management-ingress-operator
+    - name: IBM_MANAGEMENT_INGRESS_OPERATOR_IMAGE
       image: quay.io/opencloudio/ibm-management-ingress-operator
```

Comment:
shouldn't thee `:tag` be usd, for CICD to correctly replace at the build time with the SHA256 digest?

Reply:
Yes, @pgodowski . Will be updating it. Thank you!